packages/reference/src/resources/useResourceLinkActions.ts

```diff
@@ -25,39 +32,21 @@ export function useResourceLinkActions({
   );
   const multiple = field.type === 'Array';
 
-  const toLinkItem = useMemo(() => {
-    function toUrn(entry: EntryProps<KeyValueMap>) {
-      return `crn:${apiUrl}:::content:spaces/${entry.sys.space.sys.id}/entries/${entry.sys.id}`;
-    }
-
-    return (entry: EntryProps<KeyValueMap>) => {
-      return {
-        sys: {
-          type: 'ResourceLink',
-          linkType: 'Contentful:Entry',
-          urn: toUrn(entry),
-        },
-      };
-    };
-  }, [apiUrl]);
-
   const onLinkedExisting = useMemo(() => {
-    if (multiple) {
-      return (entries: EntryProps<KeyValueMap>[]) => {
-        const linkItems = entries.map(toLinkItem);
+    return (entries: EntryProps<KeyValueMap>[]) => {
```

Comment:
I think you can kill all the `<KeyValueMap>` generics as this is the default anyway.